pegasus/sites.v3/hourofcode.com/views/resources_nav.haml

```diff
@@ -0,0 +1,13 @@
+%div{style: "margin-top: 10px;"}
+  %a{href: resolve_url('/promote')}
+    %div.display-link
+      %img.display-img{src: "/images/flyers.png"}
```

Comment:
you can try requesting a file stored as a png as a jpg and pegasus will convert it for you and varnish will cache it.  it's worth trying on photographic-style images just to see if we can save some significant bandwidth.